main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
Ah, I see ... You are very good at this. =P Um ... can we just ignore this case? XD For example, if you make a range input element and you don't specify the value attribute, then getAttribute('value') will return null. However, if you access the value property, then you will get a default value of "50". I think this behavior is consistent with at least some of the standard elements. XD